packages/flutter/test/painting/binding_test.dart

```diff
@@ -130,4 +130,9 @@ class FakeImage implements ui.Image {
 
   @override
   int get width => 10;
+
+  @override
+  dynamic noSuchMethod(Invocation invocation) {
```

Comment:
Why can the use of this FakeImage not be replaced with the new test image stuff?